x-pack/plugins/cases/common/api/connectors/mappings.ts

```diff
@@ -26,7 +26,7 @@ export type ThirdPartyField = rt.TypeOf<typeof ThirdPartyFieldRT>;
 export const ConnectorMappingsAttributesRT = rt.type({
   action_type: ActionTypeRT,
   source: CaseFieldRT,
-  target: ThirdPartyFieldRT,
+  target: rt.union([ThirdPartyFieldRT, rt.undefined]),
```

Comment:
Why it can be `undefined`?

Reply:
if they don't map the comment field. alert name is the only required mapping

Reply:
This change affects all others connector. This change is for the cases mapping and not the swimlane mapping. I think it is best to leave it mandatory for the moment and handle on the backend if they don't map the comment field.